uber/templates/regform.html

```diff
@@ -297,7 +299,7 @@
 </div>
 
 <div class="form-group">
```

Comment:
We now make this bold by default and it gets unbolded when the checkbox is unchecked (the function which toggles this also runs by default when the page loads).  I've tested this on several of the forms to make sure it works the way we expect.